src/careamics/cli/conf.py

```diff
@@ -0,0 +1,404 @@
+"""Configuration building convinience functions for the CAREamics CLI."""
+
+import os
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Tuple
+
+import click
+import typer
+import yaml
+from typing_extensions import Annotated
+
+# import hydra
+from ..config import (
+    Configuration,
+    create_care_configuration,
+    create_n2n_configuration,
+    create_n2v_configuration,
+)
+
+WORK_DIR = Path(os.getcwd())
+
+app = typer.Typer()
+
+
+def _save_config(config: Configuration, fp: Path) -> None:
```

Comment:
Why not use `careamics:save_configuration` and ignore its output?
https://github.com/CAREamics/careamics/blob/b7c21ce30ac5e8ff0f7e3e30c453c44c0e60ab58/src/careamics/config/configuration_model.py#L551

Reply:
I didn't know this existed. I will change it